src/cli/telemetry/test/helpers.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { tryFn, uuidv4 } from '../helpers';
+
+describe('uuidv4', () => {
+
+	it('should output a UUID', async () => {
+		const pattern = new RegExp(/^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i);
+		const uuid = uuidv4()
+		expect(!!uuid.match(pattern)).toBe(true)
+	});
+
+});
+
+describe('tryFn', () => {
+
+	it('should handle failures correctly', async () => {
+		const result = await tryFn(async () => {
+			throw new Error("Uh oh!")
+		});
+
+		expect(result).toBe(null)
+	});
+
+	it('should handle success correctly', async () => {
+		const result = await tryFn(async () => {
+			return true
+		})
+
+		expect(result).toBe(true)
+	});
+	it('handles returning false correctly', async () => {
```

Comment:
Non blocking: It'd be nice to have the spacing of the tests here be the same, since Prettier won't enforce that for us (I think)
```suggestion

	it('handles returning false correctly', async () => {
```